packages/daisy/src/components/tabs/tab.tsx

```diff
@@ -16,7 +16,7 @@ export const Tab = component$(
         class={`tab ${isBordered ? 'tab-bordered' : ''} ${
           isLifted ? 'tab-lifted' : ''
         }`}
-        selectedClassName="tab-active"
+        selectedclass="tab-active"
```

Comment:
I think that in this case, it makes sense to call it `selectedClassName` (just to make sure the prop name is clear enough that it's not a ES Class or something)

WDYT?

Reply:
I would go for `classActive`.

```tsx
<Tab class={/* ... */} classActive="tab-active"/>
```

To me, this looks convenient.

Reply:
Ha, just noticed, that this came from my search and replace.

I should revert this, and we can have a discussion on API-Surface for components later.